### DISCUSSION TITLE
Bloggen kan trenge en RSS-feed

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#2285] [Annet]
Hvilke steg i [prosessen](https://fremtind.github.io/jokul/komigang/prosessen) er utført?
- [x] Undersøke
- [x] Rapportere
- [ ] Avklare
- [ ] Spesifisere krav
- [ ] Utvikle
- [ ] Ferdig!

**Er ønsket tilknyttet et problem? Fortell oss mer!**
Jeg vil gjerne følge med på [Jøkul-bloggen](https://jokul.fremtind.no/blog/auto-dark-mode/) og bli varslet om nye bloggposter slik som andre blogger. Notifications på Github er liksom ikke helt det samme 😉 

**Beskriv løsningen du ønsker**
Jeg ønsker meg en RSS-feed jeg kan abonnere på. Gatsby har f. eks. [denne guiden](https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-an-rss-feed/) vi kan ta utgangspunkt i. Denne feeden kan gjerne nevnes og lenkes til inne på blogg-siden, men bør i det minste lenkes til i  `head` for at [ymse RSS-lesere skal finne fram](https://www.petefreitag.com/item/384.cfm).

```html
<link rel="alternate" type="application/rss+xml"  title="Blogg | Jøkul designsystem" href="/path-til-feed/" />
```

**Comment by piofinn** [comment]
> RSS-feed

![long_time_star_wars](https://user-images.githubusercontent.com/25739615/133266947-81cc21a9-2290-4fce-94ef-4314fd956546.gif)

**Comment by wkillerud** [comment]
RSS-brukere:

![](https://media.giphy.com/media/kSlJtVrqxDYKk/giphy-downsized.gif?cid=ecf05e47kdbchbc9886pvbk12jmlnv0xgm2owuariufbuupu&rid=giphy-downsized.gif&ct=g)

**Reply by joms** [reply]
![Kill it with fire](https://media3.giphy.com/media/BTbo1iT1yEfOE/giphy.gif?cid=ecf05e47f2toodavvo0e5zfa9krfrtfufgzavob5mqxswv58&rid=giphy.gif&ct=g)

**Comment by wkillerud** [comment]
Løst i #2744
